core/src/apps/debug/__init__.py

```diff
@@ -418,6 +418,10 @@ async def dispatch_DebugLinkGetGcInfo(
             ]
         )
 
+    async def dispatch_RestartEventLoop(msg: RestartEventLoop) -> None:
+        """Restart the event loop."""
+        raise RestartEventLoopException
```

Comment:
Not sure if we also should send a "Success" message back before restarting the event loop (similar to `WipeDevice`) - WDYT?